src/processing/dom/index.js

```diff
@@ -166,6 +148,11 @@ export default class DomProcessor {
                 selector:          selectors.IS_SVG_ELEMENT_WITH_XLINK_HREF_ATTR,
                 urlAttr:           'xlink:href',
                 elementProcessors: [this._processSVGXLinkHrefAttr, this._processUrlAttrs]
+            },
+            {
+                selector:          selectors.IS_SVG_ELEMENT_WITH_XML_BASE_ATTR,
+                urlAttr:           'xml:base',
+                elementProcessors: [this._processUrlAttrs]
             }
         ];
```

Comment:
Can we rewrite it as (pseudo code):
1)change selector to `selector.isSVGElement`
2)set for `elementProcessors` value `[this._processSVGElement, this.processUrlAttrs]`
where  `this._processSVGElement` has code:
```javascript
function _processSVGElement (el, urlProcessor) {
     if (el.hasAttr('xml:base' || (el.hasAttr('xlink:href') && SVG_XLINK_HREF_TAGS.indexOf(el.tagName) !== -1) )  {
            ...process url
    }
}
```